### ISSUE TITLE
More extensive API docs on RTFD

### ISSUE THREAD

**Issue by rgerkin** [scidash/sciunit#68]
The API docs could use some explanatory pages before diving right into function references.

**Comment by ChihweiLHBird** [comment]
Is readthedoc used for solving this?

**Comment by rgerkin** [comment]
@ChihweiLHBird Yes, I will close this after I get a chance to review all the docs and merge into master.
